doc/ref_impl/cntt-ri2/README.md

```diff
@@ -5,19 +5,20 @@
 This is the Kubernetes-based Reference Implementation (RI-2)
 
 ## Release Information
-**Bundle: _6.0-alpha_**
+**Bundle: _6.0_**
```

Comment:
Sorry for missing the call. I assume you discussed whether to go for 6.0 or 7.0? We switched from 4.0-alpha to 6.0-alpha as well without a 4.0 release. Just curious.

Reply:
@gkunz Yes we discussed and as the RA2 LAkelse was labeled 7.0 and RI2 is a version behind we settled on 6.0

Reply:
sounds good to me.